params.yml

```diff
@@ -179,3 +182,20 @@ args:
 
   ## Tools versions
   polycli_version: v0.1.42
+
+  ## blutgang configuration.
+  blutgang_image: makemake1337/blutgang:latest
+
+  blutgang_http_address: 0.0.0.0:55555
+  blutgang_admin_address: 0.0.0.0:55556
+  blutgang_rpc_port: 55555
+  blutgang_admin_port: 55556
+
+  l2_sequencer_url: http://zkevm-node-sequencer-001:8123
+  l2_sequencer_ws_url: ws://zkevm-node-sequencer-001:8133
+
+  l2_rpc_url: http://zkevm-node-rpc-001:8123
+  l2_ws_url: ws://zkevm-node-rpc-001:8133
+
+  l2_rpc_pless_url: http://zkevm-node-rpc-pless-001:8123
+  l2_ws_pless_url: ws://zkevm-node-rpc-pless-001:8133
```

Comment:
Those can be retrieved directly in the code using `plan.get_service()`.

For example for the sequencer url, you can have the following piece of code:

```bash
$ kurtosis enclave inspect cdk-v1
zkevm-node-sequencer-001                                                RUNNING
    data-streamer: 6900/tcp -> datastream://127.0.0.1:62470
    pprof: 6060/tcp -> http://127.0.0.1:62469                 
    prometheus: 9091/tcp -> http://127.0.0.1:62472            
    rpc: 8123/tcp -> http://127.0.0.1:62471   
...                
```

```starlark
zkevm_sequencer_service = plan.get_service("zkevm-node-sequencer"+args["deployment_suffix"])
zkevm_sequencer_url = "http://{}:{}".format(zkevm_sequencer_service.ip_address, zkevm_sequencer_service.ports["rpc"].number)
```

Reply:
Thanks for the heads up! I also think it would make sense to fetch it directly using `plan.get_service()` if this was being used in the `.star` file, but the variables are being used inside the `config.toml` file. Not 100% sure if this would work in this case as well..

Reply:
Here is an example for this: https://github.com/0xPolygon/kurtosis-cdk/pull/87/files#r1579154651